ls8/cpu.c

```diff
@@ -29,7 +29,7 @@ int cpu_load(struct cpu *cpu, int argc, char **argv)
         return 1;
     }
 
-    fp = fopen(argv[1], "r");
+    fp = fopen(argv, "r");
```

Comment:
hmmm, is this working? because what you deleted before looks right

Reply:
yeah the naming convention I used here might be a little misleading. The type of the argument I'm passing in is char **, so argv actually refers to one of the items in argv. So when this function is called in ls8.c, argv[1] is passed in there. It does kind of make more sense to just make this argument a char * and pass in the full argv array and specify the index inside the function.